matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/tasks/SendToDeviceTask.kt

```diff
@@ -46,16 +46,21 @@ internal class DefaultSendToDeviceTask @Inject constructor(
                 messages = params.contentMap.map
         )
 
+        // Create a unique txnId first, to use the same value if the request is retried
```

Comment:
This comment is a bit hard to parse, also isn't it the other way around? We first try an existing transaction ID and if there isn't one we create a new random one?

Reply:
I agree, I will update it